datalake/src/main/java/com/sequenceiq/datalake/entity/SdxCluster.java

```diff
@@ -138,8 +134,36 @@ public class SdxCluster implements AccountAwareResource {
     @Column(nullable = false)
     private boolean detached;
 
+    /**
+     * @deprecated Kept only for backward compatibility. Use the {@link SdxDatabase#createDatabase} instead. Will be removed in CB-21369.
+     */
+    @Deprecated
+    @Column(nullable = false)
+    private boolean createDatabase;
+
+    /**
+     * @deprecated Kept only for backward compatibility. Use the {@link SdxDatabase#databaseCrn} instead. Will be removed in CB-21369.
+     */
+    @Deprecated
+    private String databaseCrn;
+
+    /**
+     * @deprecated Kept only for backward compatibility. Use the {@link SdxDatabase#databaseEngineVersion} instead. Will be removed in CB-21369.
+     */
+    @Deprecated
     private String databaseEngineVersion;
 
+    /**
+     * @deprecated Kept only for backward compatibility. Use the {@link SdxDatabase#databaseAvailabilityType} instead. Will be removed in CB-21369.
+     */
+    @Deprecated
+    @Convert(converter = SdxDatabaseAvailabilityTypeConverter.class)
+    private SdxDatabaseAvailabilityType databaseAvailabilityType;
+
+    @OneToOne(cascade = CascadeType.REMOVE, orphanRemoval = true)
```

Comment:
It's not working, because SdxDatabase contains sdxcluster_id which cannot be null. In the next cb version it can be changed when sdxcluster_id will be removed, there is a followup jira ticket for this: [CB-21369](https://jira.cloudera.com/browse/CB-21369)